AuctoMarket/BuyerMarketplace
# feat(AUCTO-86): update Payment Completed Page UI (mobile)

Jira: AUCTO-86

## Synopsis

update Payment Page UI (mobile)

## Changes

- [x] Implement the task

## How to Test

Check Netlify preview link https://deploy-preview-51--aucto.netlify.app/orders/test/payment-status
